<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exchange_popup</class>
 <widget class="QDialog" name="Exchange_popup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_remain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Points remain after exchange:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_B">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mode">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Points : Cash</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exchange_rate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20 : 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_earned">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cash earned after exchange:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Exchange Rate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exchange</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
